library/include/hipblaslt-ext.hpp

```diff
@@ -117,6 +120,56 @@ namespace hipblaslt_ext
             = 0; //!< The aux batch stride. Only works if mode is set to aux related epilogues.
     };
 
+    /*! \ingroup types_module
+     *  \brief hipblasLt extension Epilogue for gemm problems.
+     *
+     * \details This class sets the epilogue of a gemm problem.
+     */
+    class GemmEpilogueV2
+    {
+    public:
+        HIPBLASLT_EXPORT GemmEpilogueV2();
+        HIPBLASLT_EXPORT ~GemmEpilogueV2();
+
+        HIPBLASLT_EXPORT                 GemmEpilogueV2(const GemmEpilogueV2& epilogue);
+        HIPBLASLT_EXPORT GemmEpilogueV2& operator=(const GemmEpilogueV2& epilogue);
+
+        HIPBLASLT_EXPORT                 GemmEpilogueV2(GemmEpilogueV2&& epilogue);
+        HIPBLASLT_EXPORT GemmEpilogueV2& operator=(GemmEpilogueV2&& epilogue);
+
+        HIPBLASLT_EXPORT void
+            setMode(hipblasLtEpilogue_t mode); //!< Set the mode of epilogue. Default is gemm.
+        HIPBLASLT_EXPORT void setBiasDataType(
```

Comment:
It is better to return status for SET APIs

Reply:
SUCCUESS is the only option, is this necessary?